WordPress/src/main/java/org/wordpress/android/ui/JetpackRemoteInstallViewModel.kt

```diff
@@ -55,7 +54,6 @@ private const val EMPTY_MESSAGE = "EMPTY_MESSAGE"
 
 class JetpackRemoteInstallViewModel
 @Inject constructor(
-    private val jetpackStore: JetpackStore,
```

Comment:
You cannot remove this because without the injected Store the FluxC doesn't accept emitted events the remote install is broken.

Reply:
Fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/9694/commits/6a53d6914ef1b3bd2a5ef1c837d6bdb03666ab04 and suppressed the warning in https://github.com/wordpress-mobile/WordPress-Android/pull/9694/commits/ac0a5b567c5adb7f7eb1558442239e4ea5cced79